nexteuropa_newsroom.module

```diff
@@ -673,16 +668,68 @@ function nexteuropa_newsroom_feeds_after_save(FeedsSource $source, $entity, $ite
           }
 
           // If title is empty it means there is no translation.
-          if (!empty($xpath_expression) && isset($item[$xpath_expression]) && empty($item[$xpath_expression])) {
+          if (!empty($xpath_expression) || empty($item[$xpath_expression])) {
             NexteuropaNewsroomHelper::deleteEntityTranslations($entity, $entity_id, $language);
           }
         }
       }
     }
 
-    _nexteuropa_newsroom_set_image_meta_info('field_newsroom_illustrative_img', $entity, $source, $item, $languages, NexteuropaNewsroomHelper::getDefaultLanguage());
-    _nexteuropa_newsroom_set_image_meta_info('field_newsroom_private_img', $entity, $source, $item, $languages, NexteuropaNewsroomHelper::getDefaultLanguage());
+    $image_fields = [
```

Comment:
root > test-phpcs-exec:
...............W..................E......................... 60 / 63 (95%)
...

FILE: ...ures/nexteuropa-newsroom-reference/nexteuropa_newsroom.forms.inc
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
 396 | WARNING | Unused variable $guid.

FILE: ...eatures/nexteuropa-newsroom-reference/nexteuropa_newsroom.module
FOUND 3 ERRORS AND 2 WARNINGS AFFECTING 5 LINES
  680 | WARNING | [x] A comma should follow the last multiline array
      |         |     item. Found: 'field_newsroom_private_img'
  730 | ERROR   | [x] Functions must not contain multiple empty lines
      |         |     in a row; found 2 empty lines
 1253 | WARNING | [x] A comma should follow the last multiline array
      |         |     item. Found: 'field_newsroom_private_img'
 1319 | ERROR   | [x] Array indentation error, expected 6 spaces but
      |         |     found 10
 1320 | ERROR   | [x] Array closing indentation error, expected 4
      |         |     spaces but found 8

PHPCBF CAN FIX THE 5 MARKED SNIFF VIOLATIONS AUTOMATICALLY

Time: 29.61 secs; Memory: 119.5Mb